huaweicloud/services/cbr/resource_huaweicloud_cbr_policy.go

```diff
@@ -136,14 +136,15 @@ func ResourceCBRPolicyV3() *schema.Resource {
 						},
 					},
 				},
-				RequiredWith: []string{"backup_quantity", "time_zone"},
+				RequiredWith: []string{"backup_quantity"},
 			},
 			"time_zone": {
 				Type:     schema.TypeString,
 				Optional: true,
 				Computed: true,
 				ValidateFunc: validation.StringMatch(regexp.MustCompile(`^UTC[+-]\d{2}:00$`),
 					"The time zone must be in UTC format, such as 'UTC+08:00'."),
+				RequiredWith: []string{"long_term_retention"},
```

Comment:
to keep compatibility, you can deal it in Read func.

Reply:
done.